src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -34,7 +44,7 @@ class FileLinkFormatter
      */
     public function __construct($fileLinkFormat = null, RequestStack $requestStack = null, string $baseDir = null, $urlFormat = null)
     {
-        $fileLinkFormat = $fileLinkFormat ?: ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format');
+        $fileLinkFormat = (self::FORMATS[$fileLinkFormat] ?? $fileLinkFormat) ?: ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format');
```

Comment:
passing `null` as an array key won't work

Reply:
Are you sure about that? From https://www.php.net/manual/en/language.types.array.php#example-58

> Null will be cast to the empty string, i.e. the key null will actually be stored under "".